### ISSUE TITLE
Phalcon classes not found

### ISSUE THREAD

**Issue by marcotroisi** [scrutinizer-ci/php-analyzer#572]
The analyzer keep saying that *Phalcon* classes do not exist, when they evidently do, but they are probably not being found because Phalcon is a C extension.

see https://scrutinizer-ci.com/g/rocket-internet-berlin/skyrocket/issues/develop/files/app/Common/Form/Validator/NotExists.php?orderField=path&order=asc&issueId=f91afb99-fc8e-4c60-89a5-9f40356576eb

**Comment by schmittjoh** [comment]
Indeed, we do not have special support for this atm.

Are you using something like https://github.com/phalcon/phalcon-devtools/tree/master/ide for IDE auto-completion already?

**Comment by marcotroisi** [comment]
Yes, I am

**Comment by schmittjoh** [comment]
Is this part of your repository, or do you have this locally somewhere?

**Comment by marcotroisi** [comment]
no, we don't include Composer's `vendor` folder, so it's only available locally...

**Comment by schmittjoh** [comment]
If you can add the stubs of the version that you are using to the repository, you could make the analyzer aware of these classes by adding that folder as a dependency:

```
tools:
    php_analyzer:
        dependency_paths:
            - path/to/stubs/folder
```

Unfortunately, this is the only quick solution that I can suggest. 

Alternatively, you can also ignore these issues (https://scrutinizer-ci.com/docs/reviews/filtering_issues_and_false_positives), but depending on how many different classes these are, this might not be ideal.

I'll keep this open, we might come up with something better in the long-term.

**Comment by marcotroisi** [comment]
We actually have this:

```
dependency_paths:
                   - vendor/*
```

So it looks like we are already including it in the project...
